ext/standard/tests/mail/bug80751.phpt

```diff
@@ -1,97 +1,92 @@
 --TEST--
 Bug #80751 (Comma in recipient name breaks email delivery)
 --EXTENSIONS--
-imap
---CONFLICTS--
-imap
+curl
```

Comment:
Technically, all of those tests also require JSON extension (which I know can't be disabled now; but who knows if this will always be the case)

Reply:
Thanks, I overlooked it

Reply:
we don't set json as it cannot be disabled (we actually removed it from EXTENSIONS and skip parts in all tests when to got always enabled).

Reply:
Did we actually remove them? I don't really see any harm in having it in the EXTENSIONS section as that section is mainly used for Windows to load shared extensions if available and skip if not.

Reply:
Yeah think it got removed but tests. It's not shared extension for Windows either so there is no point to specify it. It's like adding standard or spl there...